tests/models/mask2former/test_modeling_mask2former.py

```diff
@@ -481,3 +482,28 @@ def test_with_segmentation_maps_and_loss(self):
             outputs = model(**inputs)
 
         self.assertTrue(outputs.loss is not None)
+
+    def test_export(self):
+        if version.parse(torch.__version__) < version.parse("2.4.0"):
+            self.skipTest(reason="This test requires torch >= 2.4 to run.")
```

Comment:
Can you please update 

```python
from transformers.pytorch_utils import is_torch_greater_or_equal_2_4
```
instead of doing comparison with packaging